src/constants/tokens/testNetRskCrossBinance.js

```diff
@@ -1,6 +1,11 @@
 import * as tokensInfo from '../tokensInfo'
 import * as methodType from '../methodType'
 
+export const TEST_NET_RSK_CROSS_BINANCE_GAS_TOKEN = {
+  ...tokensInfo.TOKEN_RBTC_INFO,
+  symbol: 'rBTC',
```

Comment:
When grossing form RSK Testnet to Binance, the gas on the binance network is BNB so it should be
`symbol: BNB`

Reply:
this is used when crossing from Binance to RSK, so it's rBTC.

Reply:
you are correct